janus_core/cli/utils.py

```diff
@@ -17,6 +17,22 @@
 from janus_core.helpers.utils import dict_remove_hyphens
 
 
+def set_read_kwargs_index(read_kwargs: dict[str, Any]) -> None:
+    """
+    Set default read_kwargs["index"] and check validity.
+
+    Parameters
+    ----------
+    read_kwargs : dict[str, Any]
+        Keyword arguments to be passed to ase.io.read.
+    """
+    read_kwargs.setdefault("index", 0)
```

Comment:
`":"` is still a valid ASE index, but is invalid here. Need a comment to that effect?

Reply:
Good point, thanks